pkg/cmd/workspace/code.go

```diff
@@ -70,7 +71,7 @@ var CodeCmd = &cobra.Command{
 
 			workspace, err := apiclient_util.GetWorkspace(encodedWorkspaceName)
 			if err != nil {
-				if errors.Is(err, workspaces.ErrWorkspaceNotFound) {
+				if strings.Contains(err.Error(), workspaces.ErrWorkspaceNotFound.Error()) {
```

Comment:
we need to use strings to compare because the error can't be unwrapped as it is handled by many other functions along with api. also i tried using errors.Is but it did not worked as expected

Reply:
![image](https://github.com/user-attachments/assets/48fe58e0-effb-4129-bc9b-eee3330c634f)

Reply:
Still it is not feasible to have this `DEBU[0000]` line.
I managed to get the same output. Let's get rid of that and then we'll proceed.

Reply:
ok

Reply:
done. removed log.debug(err)

Reply:
@Divanshu-Grover I'm so sorry for this inconvenience, my colleague Toma informed me of my mistake.
Can you please revert this log.Debug removal as it is need as Toma said in one of the comments above?
Once again, sorry for the mistake :)